work_queue/src/work_queue_json.h

```diff
@@ -0,0 +1,19 @@
+#ifndef WORK_QUEUE_JSON_H
+#define WORK_QUEUE_JSON_H
+
+#include "work_queue.h"
+#include "jx.h"
+
+int is_in(const char* str, char* a[]);
```

Comment:
Since these are private functions, don't mention them in the public header.
Prototype them at the top of `work_queue_json.h` if necessary.